composer.json

```diff
@@ -24,7 +24,9 @@
     },
     "require-dev": {
         "league/oauth2-facebook": "^1.1",
-        "symfony/security-guard": "^2.8|^3.0"
+        "symfony/security-guard": "^2.8|^3.0",
+        "phpunit/phpunit": "^4.8",
```

Comment:
It would also need finetuning of the `php-code-coverage` since they're intertwined, but yeah it could work. In general though I myself favor just picking a single major to avoid having differences between developers. I would just postpone the 5.x upgrade until PHP 5.5 is dropped in this project itself.